src/Orchard.Web/Modules/Orchard.OpenId/Controllers/AccountController.cs

```diff
@@ -46,6 +46,7 @@ public class AccountController : Controller {
         public ILogger Logger { get; set; }
         public Localizer T { get; set; }
 
+        [AlwaysAccessible]
```

Comment:
To consistent to Orchard.User module, and also to defend some condition changes in future,
[AlwaysAccessible] can still work inside whole orchard system